docs/sources/flow/getting-started/migrating-from-promtail.md

```diff
@@ -0,0 +1,210 @@
+   the converted configuration may not match the original Promtail
+   configuration. Do not use this flag in a production environment.{{%
+   /admonition %}}
+
+## Example
+
+This example demonstrates converting a Promtail configuration file to a Grafana
+Agent Flow Mode configuration file.
+
+The following Promtail configuration file provides the input for the conversion:
+
+```yaml
+clients:
+  - url: http://localhost/loki/api/v1/push
+scrape_configs:
+  - job_name: example
+    static_configs:
+      - targets:
+          - localhost
+        labels:
+          __path__: /var/log/*.log
+```
+
+The convert command takes the YAML file as input and outputs a River file.
+
+```bash
+AGENT_MODE=flow; grafana-agent convert --source-format=promtail --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+```
+
+The new Flow Mode configuration file looks like this:
+
+```river
+local.file_match "example" {
+	path_targets = [{
+		__address__ = "localhost",
+		__path__    = "/var/log/*.log",
+	}]
+}
+
+loki.source.file "example" {
+	targets    = local.file_match.example.targets
+	forward_to = [loki.write.default.receiver]
+}
+
+loki.write "default" {
+	endpoint {
+		url = "http://localhost/loki/api/v1/push"
+	}
+	external_labels = {}
+}
+```
+
+## Limitations
+
+Config conversion is done on the best-effort basis; the Agent will issue
+warnings or errors where the conversion cannot be performed.
+
+Once the configuration is converted, it is recommended for users to understand
+and review the output Flow Mode configuration and verify that it is correct
+before starting to use it in a production environment.
+
+Furthermore, it is recommended to go through the following checklist:
+
+* Check if you are using any extra command line arguments with Promtail which
+  are not present in your config file, e.g. `-max-line-size`
+* Check if you are setting any environment variables -
+  whether [expanded in the config file][] itself or consumed directly by
+  Promtail such as `JAEGER_AGENT_HOST`.
+* In Flow Mode, the positions file will be saved at a different location -
+  see [loki.source.file][] for more details. Check if you have any existing
+  setup (e.g. Kubernetes Persistent Volume) that needs changing to use the new
+  positions file path.
+* Metrics exposed by the Flow Mode will usually match those of Promtail, but
+  will use a different name. Make sure that you use the new metric names, e.g.
+  in your alerts and dashboards queries.
+* Please note that the logs produced by the Agent will differ from those
+  produced by Promtail.
+* Please note that the Agent exposes [Grafana Agent Flow UI][], which differs
+  from Promtail's Web UI.
+
+[expanded in the config file]: https://grafana.com/docs/loki/latest/clients/promtail/configuration/#use-environment-variables-in-the-configuration
+
+[Grafana Agent Flow UI]: http://localhost:3003/docs/agent/latest/flow/monitoring/debugging/#grafana-agent-flow-ui
```

Comment:
check the url (localhost)

Reply:
Whoops, sorry! Was a bit in a rush when I pushed this. Will clean up now.